docs/modules/ROOT/pages/documentation/extensions/grpc/README.adoc

```diff
@@ -0,0 +1,344 @@
+[[grpc]]
+= gRPC
+
+[[what-is-grpc]]
+== What is gRPC?
+
+The gRPC is a modern open-source high performance Remote Procedure Call (RPC) framework.
+
+[[what-can-we-do-with-it]]
+== What can we do with it?
+
+* We can create services more efficient than REST.
+* Use HTTP2 to provide full duplex communication between the client and the server.
+
+The gRPC support provide the means for users to use the gRPC style service programming.
+
+You can find detailed information about gRPC here: https://grpc.io/docs/what-is-grpc/introduction/.
+
+[[installing-grpc-support]]
+== Installing gRPC Support
+
+You need to clone the following repo on your environment: https://github.com/payara/gRPC
+
+After cloning you need to build and drop the JAR from the target generated files into the `${PAYARA_HOME}/glassfish/modules` directory of your Payara Server Installation.
```

Comment:
I tryied to build from master branch, but a dependency is missing:
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.101 s
[INFO] Finished at: 2022-03-03T09:36:22-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project grpc-support: Could not resolve dependencies for project fish.payara.extensions.grpc:grpc-support:jar:1.0.0: The following artifacts could not be resolved: fish.payara.server.internal.common:glassfish-api:jar:5.2020.5, fish.payara.server.internal.common:internal-api:jar:5.2020.5, fish.payara.server.internal.deployment:dol:jar:5.2020.5: fish.payara.server.internal.common:glassfish-api:jar:5.2020.5 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :grpc-support